### ISSUE TITLE
Cockroachdb 1.1.0 compatibility

### ISSUE THREAD

**Issue by jorgebarredo** [heroiclabs/nakama#121]
## Description

Error throw when trying to add a friend using CockroachDB 1.1.0

## Steps to Reproduce

1. Edit docker-compose.yml to change cockroachdb version to 1.1.0
2. Type docker-compose up
3. Wait until server is running.
3. User A send a friend request to user B
4. Errors reported to client and in server logs

## Expected Result
No error is thrown and friend request is processed.

## Actual Result
Error thrown "Could not add friend". Server prints out the following stacktrace:
`"nakama/vendor/go.uber.org/zap.Stack\n\tvendor/go.uber.org/zap/field.go:209\nnakama/vendor/go.uber.org/zap.(*Logger).check\n\tvendor/go.uber.org/zap/logger.go:273\nnakama/vendor/go.uber.org/zap.(*Logger).Error\n\tvendor/go.uber.org/zap/logger.go:176\nnakama/server.(*pipeline).friendAddById\n\tserver/pipeline_friend.go:321\nnakama/server.(*pipeline).friendAdd\n\tserver/pipeline_friend.go:295\nnakama/server.(*pipeline).processRequest\n\tserver/pipeline.go:114\nnakama/server.(*pipeline).(nakama/server.processRequest)-fm\n\tserver/session_auth.go:164\nnakama/server.(*session).Consume\n\tserver/session.go:100\nnakama/server.(*SessionRegistry).add\n\tserver/session_registry.go:80\nnakama/server.(*authenticationService).configure.func4\n\tserver/session_auth.go:164\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/Cellar/go/1.9/libexec/src/net/http/server.go:1918\nnakama/vendor/github.com/gorilla/mux.(*Router).ServeHTTP\n\tvendor/github.com/gorilla/mux/mux.go:103\nnakama/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP\n\tvendor/github.com/gorilla/handlers/cors.go:51\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/Cellar/go/1.9/libexec/src/net/http/server.go:2619\nnet/http.(*conn).serve\n\t/usr/local/Cellar/go/1.9/libexec/src/net/http/server.go:1801"`

## Context
- [X] Unity
- [ ] Unreal
- [ ] Other

## Your Environment
- Nakama: 1.0.2 (1.0.2+708da56)
- Cockroach: 1.1.0
- Environment name and version: Docker version 17.09.0-ce
- Operating System and version: Ubuntu 16.04.3 LTS

Running nakama and cockroachdb using docker with the docker-compose.yml

**Comment by Miwwa** [comment]
## Description
`RuntimeException` Error thrown when Unity client fetch leaderboard records with cursor

## Steps to Reproduce
Unity code example:
```
var messageBuilder = new NLeaderboardRecordsListMessage.Builder(id);
client.Send(message.Build(), (INResultSet<INLeaderboardRecord> list) => {
  INCursor cursor = list.Cursor;
  if (cursor != null && list.Results.Count > 0) {
    var message = messageBuilder.Cursor(cursor).Build();

    client.Send(message, (INResultSet<INLeaderboardRecord> nextList) => {
      foreach (var r in nextList.Results) {
        Debug.LogFormat("Record handle '{0}' score '{1}'.", r.Handle, r.Score);
      }
    }, errorHandler);
  }
}, errorHandler);
```

## Expected Result
No error is thrown and results are returned by server.

## Actual Result
Server print this error:
```
{"level":"error","ts":"2017-11-17T13:26:00.598Z","msg":"Could not execute leaderboard records list query","server":"nakama-972f","uid":"e260876f-c352-4b45-8b7f-5342608f04b6","sid":"g0BiN_r-TAy5Gf-MQ-V00Q","cid":"8968b990-5cc1-4ed3-b911-9ecb0a2da434","error":"pq: could not determine data type of placeholder $5","stacktrace":"nakama/vendor/go.uber.org/zap.Stack\n\tvendor/go.uber.org/zap/field.go:209\nnakama/vendor/go.uber.org/zap.(*Logger).check\n\tvendor/go.uber.org/zap/logger.go:273\nnakama/vendor/go.uber.org/zap.(*Logger).Error\n\tvendor/go.uber.org/zap/logger.go:176\nnakama/server.leaderboardRecordsList\n\tserver/core_leaderboard.go:219\nnakama/server.(*pipeline).leaderboardRecordsList\n\tserver/pipeline_leaderboard.go:356\nnakama/server.(*pipeline).processRequest\n\tserver/pipeline.go:190\nnakama/server.(*pipeline).(nakama/server.processRequest)-fm\n\tserver/session_auth.go:128\nnakama/server.(*wsSession).Consume\n\tserver/session_ws.go:148\nnakama/server.(*SessionRegistry).addWS\n\tserver/session_registry.go:82\nnakama/server.(*authenticationService).configure.func5\n\tserver/session_auth.go:193\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/Cellar/go/1.9.2/libexec/src/net/http/server.go:1918\nnakama/vendor/github.com/gorilla/mux.(*Router).ServeHTTP\n\tvendor/github.com/gorilla/mux/mux.go:103\nnakama/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP\n\tvendor/github.com/gorilla/handlers/cors.go:51\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/Cellar/go/1.9.2/libexec/src/net/http/server.go:2619\nnet/http.(*conn).serve\n\t/usr/local/Cellar/go/1.9.2/libexec/src/net/http/server.go:1801"}
```

## Context
- [x] Unity
- [ ] Unreal
- [ ] Other

## Your Environment
- Nakama: 1.2.0
- Cockroach: 1.0.6
- Unity client: 0.9
- Operating System and version: Ubuntu server 16.04

**Comment by zyro** [comment]
This has been resolved in the [1.3.0 release](https://github.com/heroiclabs/nakama/releases/tag/v1.3.0).
